klippy/extras/palette2.py

```diff
@@ -563,12 +579,14 @@ def _run_Read(self, eventtime):
 
     def _run_Heartbeat(self, eventtime):
         self.write_queue.put(COMMAND_HEARTBEAT)
+        self.reactor.pause(5)
```

Comment:
Incorrect usage of `reactor.pause()` here.

-Kevin

Reply:
Should be the last one.